### PR TITLE
DM-29744: Rename two tables as asked by CET

### PR DESCRIPTION
**Pull request by hsinfang**

### CODE REVIEW

File: config/dp01_dc2_catalogs/idx_dpdd_forced.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "database": "dp01_dc2_catalogs",
+    "table": "dpdd_forced",
```
Comment (boutigny) [inline config/dp01_dc2_catalogs/idx_dpdd_forced.json]: Should it be "forced_photometry" ?

File: config/dp01_dc2_catalogs/idx_dpdd_ref.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "database": "dp01_dc2_catalogs",
+    "table": "dpdd_ref",
```
Comment (boutigny) [inline config/dp01_dc2_catalogs/idx_dpdd_ref.json]: Should it be "reference" ?

File: config/dp01_dc2_catalogs/idx_dpdd_forced.json (hunk 1 above)
Comment (hsinfang) [reply config/dp01_dc2_catalogs/idx_dpdd_forced.json]: Yes it should. I think it's corrected in a newer commit. Let me double check.

Comment (hsinfang) [reply config/dp01_dc2_catalogs/idx_dpdd_forced.json]: Yes, it's in https://github.com/in2p3-dp0/qserv-dataloader/pull/5/commits/54979e4df17d3a719d653ed937fda92c64b64512

Comment (boutigny) [reply config/dp01_dc2_catalogs/idx_dpdd_forced.json]: Ah yes, sorry

**Pull request merged**